File: Ejercicio4.java
public class Ejercicio4 {
    public static void main(String[] args) {
        int[][] matriz = {
            {10, 20, 30},
            {40, 50, 60},
            {70, 80, 90}
        };

        System.out.println("Matriz");
        for (int i = 0; i < matriz.length; i++) {
            for (int j = 0; j < matriz[i].length; j++) {
                System.out.print(matriz[i][j] + " ");
            }
            System.out.println();
        }

        System.out.println("Horizontal");
        for (int i = 0; i < matriz.length; i++) {
            for (int j = 0; j < matriz[i].length; j++) {
                System.out.println("[" + i + "][" + j + "]: " + matriz[i][j]);
            }
        }

        System.out.println("Vertical");
        for (int j = 0; j < matriz[0].length; j++) {
            for (int i = 0; i < matriz.length; i++) {
                System.out.println("[" + i + "][" + j + "]: " + matriz[i][j]);
            }
        }
    }
}

File: Ejercicio4.cpp
#include <iostream>
using namespace std;

int main() {
    int matriz[3][3] = {
        {10, 20, 30},
        {40, 50, 60},
        {70, 80, 90}
    };

    cout << "Matriz" << endl;
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < 3; j++) {
            cout << matriz[i][j] << " ";
        }
        cout << endl;
    }

    cout << "Horizontal" << endl;
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < 3; j++) {
            cout << "[" << i << "][" << j << "]: " << matriz[i][j] << endl;
        }
    }

    cout << "Vertical" << endl;
    for (int j = 0; j < 3; j++) {
        for (int i = 0; i < 3; i++) {
            cout << "[" << i << "][" << j << "]: " << matriz[i][j] << endl;
        }
    }

    return 0;
}
